client/src/utils/parser/robotCodeToSsotParsing/robotCodeToSsotParsing.test.js

```diff
@@ -0,0 +1,103 @@
+/* eslint-disable import/first */
+import { correctSettingsSection, correctTaskSection, taskAndApplicationCombinations, correctElementsArray, correctInstructionBlocks } from './robotCodeTestSnippets';
+
+jest.mock('../../notificationUtils');
+
+import customNotification from '../../notificationUtils';
+
+
+const parser = require('./robotCodeToSsotParsing');
+
+const ROBOT_ID = '54ab2d30eb3cc402041ac60f';
+
+sessionStorage.setItem('robotName', 'AwesomeTestRobot')
+sessionStorage.setItem('robotId', '12345678')
+sessionStorage.setItem('availableApplications', '["Excel.Application","Excel.Files","HTTP"]')
+sessionStorage.setItem('TaskApplicationCombinations', JSON.stringify(taskAndApplicationCombinations))
+
+const correctRobotCode = `${correctSettingsSection}\n\n${correctTaskSection}`
+const robotCodeAsArray = parser.getRobotCodeAsArray(correctRobotCode);
+const settingsSectionAsArray = parser.getRobotCodeAsArray(correctSettingsSection);
+const taskSectionAsArray = parser.getRobotCodeAsArray(correctTaskSection);
+const declaredApplications = parser.getApplicationArray(settingsSectionAsArray);
+
+describe('RobotCode to SSOT Parsing Tests', () => {
+    test('ssot contains right robotName', () => {
+        const ssot = parser.parseRobotCodeToSsot(correctRobotCode);
+        return expect(ssot).toHaveProperty('robotName', 'AwesomeTestRobot');
+    });
+    test('ssot contains right robotId', () => {
+        const ssot = parser.parseRobotCodeToSsot(correctRobotCode);
+        return expect(ssot).toHaveProperty('_id', 12345678);
+    });
+    test('ssot contains right starterId', async () => {
+        sessionStorage.setItem('idCounter', '5416');
+        const ssot = parser.parseRobotCodeToSsot(correctRobotCode);
+        return expect(ssot).toHaveProperty('starterId', "Event_0ay5417");
+    });
+
+    test('settings selector was found correctly', () => expect(parser.getLineNumberForSelector(settingsSectionAsArray, '*** Settings ***')).toBe(0));
+    test('settings selector was found correctly2', () => {
+        const settingsSectionAsArrayForTest = parser.getRobotCodeAsArray(`RandomOffsetLine\n${correctSettingsSection}`);
+        return expect(parser.getLineNumberForSelector(settingsSectionAsArrayForTest, '*** Settings ***')).toBe(1)
+    });
+    test('settings selector wasn\'t found (what is as expected)', () => {
```

Comment:
I don't understand the test title here

Reply:
In this test we check, that the Settings Selector wasn't found. We removed the Settings selector, therefore it is expected, that the function returns undefined here.
I've just fixed the typo. Is the test title understandable now?
```suggestion
    test('settings selector wasn\'t found (what is expected)', () => {
```

Reply:
How about 'settings selector wasn\'t found, expected to throw error' ?

Reply:
Nice Name! I've renamed it :)